src/presenter/event-presenter.js

```diff
@@ -40,18 +40,23 @@ export default class EventPresenter {
   }
 
   resetView() {
-    if (this.#mode === Mode.EDIT) {
+    if (this.#mode !== Mode.DEFAULT) {
+      this.#editEventView.reset(this.#event);
       this.#switchToViewMode();
     }
```

Comment:
Это не ошибка, просто кодстайл другой, когда выделяется early return (т.е. кейс(ы) при которых функцию просто не выполняется) в начале, а основная логика отдельно

```suggestion
    if (this.#mode === Mode.DEFAULT) {
      return;
    }

    this.#editEventView.reset(this.#event);
    this.#switchToViewMode();
```